### ISSUE TITLE
PNY NVidia P2000 - C8 Plots...?

### ISSUE THREAD

**Issue by NewSunSEO** [madMAx43v3r/chia-gigahorse#157]
I am building out my farm & do not yet have a better single slot GPU I can use for the Gigahorse Farmer.  

Dell R720XD
3060Ti - Plotter GPU
PNY NVidia P2000 - Farmer GPU (Until I get an A4000)

Will this PNY NVidia P2000 even farm at all with C8 plots?  

I'm only seeing one .25 XCH so far when I run ./chia.bin farm summary , which is making me think the PNY NVidia P2000 is not able to farm compressed C8 plots at all.  Does anyone know about this?

Thank you

**Comment by NewSunSEO** [comment]
I have another farmer, running Ubuntu with a single 18TB USB drive connected to it, with uncompressed plots on it
.  I feel like that is what has farmed the rest of this Chia, but I am not sure:

![image](https://github.com/madMAx43v3r/chia-gigahorse/assets/11369854/294114ed-9b34-4593-b162-9d2f9692d2d7)

**Comment by NewSunSEO** [comment]
Hey madMAx43v3r, what do you think, does this P2000 not have enough ram for C8 plots?  I have an A4000 on its way...

**Comment by NewSunSEO** [comment]
If anyone knows about this issue I'm seeing, please let me know.  I have an A4000 on its way, but I'm wondering if the P2000 GPU with 5GB of Memory is my issue, or if I should be looking at other things.  Any input from anyone with experience with this would be greatly appreciated.

![image](https://github.com/madMAx43v3r/chia-gigahorse/assets/11369854/1e0c5c16-9004-4e5b-8d6f-6d5ad4bb4e25)

**Comment by NewSunSEO** [comment]
Two more days & it doesn't seem like anything has been farmed with that farmer / GPU.  

Thank you

**Comment by Cathayan0** [comment]
@NewSunSEO Can you check your pool wallet?

**Comment by NewSunSEO** [comment]
Will the compressed plots only hit blocks?  I'm trying to figure out what is slowly increasing the Chia in my wallet, I'm guessing its the two 18TB drives farming the uncompressed plots?  I"m going to look for a video that explains compressed / uncompressed plots more.  

Thank you

**Comment by madMAx43v3r** [comment]
>  I'm trying to figure out what is slowly increasing the Chia in my wallet,

It's from the pool payouts, but they dont count as "farmed" in "chia farm summary", because they are regular transfers.

**Comment by NewSunSEO** [comment]
Oh, ok.  I just bought an A4000 thinking the P2000 wasn't farming...  haha...  This first pool of 6 20TB drives is still plotting C8 plots in the Dell R720XD.  Then a 2nd pool of 6 20TB for more C8 plots will be next.  I have a NetApp DS4246 arriving Thursday, which will be filled with more 20TB drives for more C8 plots.  I'm just trying to figure out where the Chia is coming from, as I continue building out my farm.

Are the compressed plots bringing in "Regular transfers" or are those from the uncompressed plots?  

Thank you for your help

**Comment by Cathayan0** [comment]
> 

They will be the same. Space pool recognizes compressed plots = uncompressed plots if your node is farming,

**Comment by madMAx43v3r** [comment]
> Are the compressed plots bringing in "Regular transfers" or are those from the uncompressed plots?

They both work the exact same way.

**Comment by NewSunSEO** [comment]
Oh OK.  Thank you for confirming what I originally thought.  Do you know a way I could begin to track the XCH coming in from the compressed plots & the uncompressed plots?  I'm going to build up the compressed plots a lot more, but whenever I can find a good deal on a USB external HDD, I"m going to grab those when I find deals.  I'd like to be able to monitor that both are generating XCH, if possible.

Thank you again for your help.

**Comment by madMAx43v3r** [comment]
you would have to use a different NFT and reward address

**Comment by krecik** [comment]
I use 2 A2000 (16GB version) to farm about 1PB of C8 plots. Works gr8.

**Comment by Kruger1981** [comment]
> I use 2 A2000 (16GB version) to farm about 1PB of C8 plots. Works gr8.

16GB? do you mean 6GB?

**Comment by NewSunSEO** [comment]
Yes, I now have an A4000 in my main farmer (Dell R720XD with 12 20TB HDD's) & using two P2000 in the two harvesters (Both Dell R720XD  with 12 20TB HDD's each) I have so far.  These three Dell 720XD's are in a High Availability Cluster using Proxmox.

I'm building a dedicated plotter next & adding the NetApp DS4246 through QSFP to the main farmer with 24 20TB HDD's.  I will then daisy chain more NetApp DS4246's together & continue building my farm out over time.  Everything is C8 MadMax plots currently.

**Comment by Kruger1981** [comment]
> Hey madMAx43v3r, what do you think, does this P2000 not have enough ram for C8 plots? I have an A4000 on its way...

Also if you want to see what the GPU is doing, use the remote compute, you can point it to your local machine and see what is going on.

I presume that the GPU is on the farmer (not a separate harvester) just run the recompute server on that machine and, depending on the OS you are using just follow the instructions for the recompute thingy.

Then you will see something like:

![image](https://github.com/madMAx43v3r/chia-gigahorse/assets/105702510/0dd3c44e-cb64-4a37-879c-e34c62e61a88)

**Comment by Kruger1981** [comment]
> R720XD

that is a nice setup.
at the moment I have a T620 for plotting.
Has 2x A2000 (6GB each) 1 for plotting and one for remote compute.
1 normal PC acting as a farmer with 4 internal 16tb hdds and a hand full of external HDDs of 12tb/14tb/16tb
1 Nuc with 14 14TB external HDDs and  one yotamaster enclosure with 5x 18TB HDDs
1 HP ML310 with 4x12tb HDD and 4x 3TB SAS HDD and 2x 12TB external HDDs
6 HP Microserver Gen 7 with 4HDD each ranging from 3TB to 10TB HDDs (these are low spec CPU very good power consumption wise but need to get remote compute as they cannot pull their weight lol)
As you can see its a bit of a mess around here, I do have a old rack server (cant remember the specs) that can take 12 HDDs and 2 NetApps that take 12HDD each as well, but didn't have the time to set them up and pass the HDDs from the Micro servers onto them.

**Comment by NewSunSEO** [comment]
WOW, that's a lot of stuff!  haha.  I wanted to keep my farm consistent, but I've contemplated picking up more assorted equipment if I came across good prices.  How many plots do you have with all that?  I have two 18TB external HDD's connected to a VM running Ubuntu (That's how all this Chia stuff started for me... lol).  I just got two more 22TB external HDD's, but the VM keeps crashing when I try to create BladeBit C7 or C6 plots on it, using a 3060Ti, so I think I"m going to return these 22TB drives & just keep adding more of the 20TB SATA drives to the JBOD & expand in that direction.

**Comment by Kruger1981** [comment]
Only have 7953 plots. Hope to get to 8000 today.
If those 22TB HDD were at a decent price, consider just taking them out of the enclosure and use them as internal HDDs
Nearly all 16TB and 18TB drives I have I bought them as external HDDs, remove them from the enclosure (some were tricky to remove) and use them as internal HDDs.
By some reason they were actually cheaper than buying the same drive as internal.
As example the 1st 3 external drives I got were 16TB HDDs at the time got them for about £250 each the price for 3 internal drives was about £280 each.
Also more often than not you will find that the HDDs inside the enclosures are Exos hard drives (Seagate) and for WD drives, they are also some sort of server HDD.

I also considered going JBOD but always reluctant that 1 drive fails and then can be a bit of a headache, so using the HDDs as standalone drives, but for the smaller capacity drives, what I have been testing is to shrink the volumes.
Basically for example all drives always have a few GB free once they are full, I shrik the volume as much as possible, create 1 partition on one of the shrinked drives now a separate partition and then extend it using all the freed shrinked space from all drives.

Like:
![image](https://github.com/madMAx43v3r/chia-gigahorse/assets/105702510/564d266e-7777-418b-9b0f-f1bc3de27051)

**Comment by NewSunSEO** [comment]
That's a good amount of plots!!!  I have around 3125 plots on each R720XD.  I really like that if one of the servers goes down, one of the other two R720's will spin up the VM's on the down server & everything continues until I can fix the down server.  Plus I can migrate from one server to another & with them in a cluster, I can manage all three in the same browser tab.  Proxmox is really cool.  

The 22tb HDD's were $395 each, but I got 21% back in rewards using my Capital One card..., that's why I got them.  So after the rewards came back, they'd be about $310.  But the 20TB drives I've been buying are $280 from NewEgg.  So, with the VM crashing & not letting me get the compressed plots created, I was just going ti keep life simple & stick with what I have working.  My idea was to re plot the 18TB external drives with compressed plots & then add these 22TB drives...

Talking with you is making me want to see if I can get these compressed plots to create on this VM one more time before returning them...  haha

**Comment by Kruger1981** [comment]
Never heard of proxmox till today, but will check it out as soon I have the time.
When you say that the VM crashes while creating the plots, how are you creating the plots? Do you have them going directly to the 22TB drives?
If so, have you considered just create the plots and then copying them manually to the HDDs?
Due to some space limitations I can't have my T620 (the plotter) any where near of the harvesters so I create the plots and then copy them over to the harvesters using a couple of 1TB external SSDs and via the network.
I know, its a bit of a pain, time consuming etc etc, but hey... keeps me busy lol
